examples/dotnet/SeleniumDocs/Interactions/PrintOptionsTest.cs

```diff
@@ -76,6 +76,15 @@ public void TestShrinkToFit()
             printOptions.ShrinkToFit = true;
             bool currentShrinkToFit = printOptions.ShrinkToFit;
         }
-    }
 
+        [TestMethod]
+        public void PrintWithPrintsPageTest() 
+        {
+            WebDriver driver = new ChromeDriver();
+            driver.Navigate().GoToUrl("https://www.selenium.dev/");
+            PrintOptions printOptions = new PrintOptions();
+            PrintDocument printedPage = driver.Print(printOptions);
+            Assert.IsNotNull(printedPage.AsBase64EncodedString);
```

Comment:
Hi @RenderMichael,

Thank you for the PR.  Since the code returns base64 code, the validation should check if the substring from index 0 to 5 contains "JVBER".

Reply:
As of .NET 8, we have a neat method `Base64.IsValid`. Do we have to maintain backwards-compatibility, or would this method suffice?

Reply:
PR feedback addressed